packages/pdf/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "rpaframework-pdf"
-version = "0.10.0"
+version = "1.26.11"
```

Comment:
Because of major version bump to `1.0.0` and because people don't trust "beta" versions like these :D.